src/Parser/Monk/Windwalker/Modules/Talents/EnergizingElixir.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS';
+import SpellIcon from 'common/SpellIcon';
+import ResourceTypes from 'common/RESOURCE_TYPES';
+import StatisticBox from 'Main/StatisticBox';
+
+import Combatants from 'Parser/Core/Modules/Combatants';
+
+import Analyzer from 'Parser/Core/Analyzer';
+
+class EnergizingElixir extends Analyzer {
+  static dependencies = {
+    combatants: Combatants,
+  };
+  chiGained = 0;
+  energyGained = 0;
+  energyWasted = 0;
+  chiSaved = 0;
+  eeCasts = 0;
+
+  on_initialized() {
+    this.active = this.combatants.selected.hasTalent(SPELLS.ENERGIZING_ELIXIR_TALENT
+      .id);
+  }
+
+  on_byPlayer_cast(event) {
+    const spellId = event.ability.guid;
+    if (spellId !== SPELLS.ENERGIZING_ELIXIR_TALENT.id) {
+      return;
+    }
+    this.eeCasts += 1;
+  }
+
+  /**
+   * Calculate the amount of Energy and Chi gained and wasted
+   * for each cast event of Energizing Elixir.
+   */
+  on_toPlayer_energize(event) {
+    const spellId = event.ability.guid;
+    if (spellId === SPELLS.ENERGIZING_ELIXIR_TALENT.id) {
+      if (event.resourceChangeType === ResourceTypes.ENERGY) {
+        this.energyWasted += event.waste;
+        this.energyGained += event.resourceChange - event.waste;
+      }
+      if (event.resourceChangeType === ResourceTypes.CHI) {
+        this.chiWasted += event.waste;
+        this.chiGained += event.resourceChange - event.waste;
+      }
+    }
+  }
+
+  statistic() {
```

Comment:
For the statistics box, to keep this consistent with other specs, can you re-write it as the following:
```js
  statistic() {
    return (<StatisticBox icon={<SpellIcon id={SPELLS.ENERGIZING_ELIXIR_TALENT.id }/>}
      value={this.energyGained}
      label={(
        <dfn data-tip={`from ${this.eeCasts} Energizing Elixir Casts`}>
          Energy gained
        </dfn>
        )}
        />
      );
    }
  }
```

This just moves the casts portion to the tooltip.

Reply:
Will do, thanks!